hal/targets/hal/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_K64F/storage_driver.c

```diff
@@ -578,19 +703,43 @@ static int32_t executeCommand(struct mtd_k64f_data *context)
         /* Execution may result in failure. Check for errors */
         if (failedWithAccessError() || failedWithProtectionError()) {
             clearErrorStatusBits();
+            #if EXISTS_POSSIBILITY_OF_CONCURRENT_READ
+                __enable_irq();
```

Comment:
Yes. As I note below, you're probably not enabling them enough.

Why do you not just enable interrupts immediately after you know it's no longer busy? And re-enable before the next launchCommand?